mods/species/bayliens/_bayliens.dme

```diff
@@ -15,12 +15,12 @@
 #include "skrell\_skrell.dm"
 #include "skrell\datum\accessory.dm"
 #include "skrell\datum\blood.dm"
+#include "skrell\datum\citizenship.dm"
 #include "skrell\datum\culture.dm"
 #include "skrell\datum\descriptor.dm"
 #include "skrell\datum\emotes.dm"
 #include "skrell\datum\faction.dm"
 #include "skrell\datum\language.dm"
-#include "skrell\datum\location.dm"
```

Comment:
We might eventually want to add this back to get codex entries & ATC fluff for actual skrell places but the things that were in here are closer to citizenship than to places anyways